flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/ExpressionReducer.scala

```diff
@@ -138,8 +138,8 @@ class ExpressionReducer(
                SqlTypeName.MULTISET =>
             reducedValues.add(unreduced)
           case SqlTypeName.VARCHAR | SqlTypeName.CHAR =>
-            val escapeVarchar = StringEscapeUtils
```

Comment:
This code should consistent with `GenerateUtils.generateLiteral`.
It is merging error when merged blink to flink.